.travis.yml

```diff
@@ -25,25 +26,23 @@ before_script:
   - if [[ $DATABASE_BACKEND == 'mysql_innodb' ]]; then mysql -e 'create database pootle CHARACTER SET utf8 COLLATE utf8_general_ci;'; fi
   - if [[ $DATABASE_BACKEND == 'mysql_innodb' ]]; then mysql -e "SET GLOBAL wait_timeout = 36000;"; fi
 script:
-  - python -m compileall -q -f .
-  - make docs
-  # FIXME revert breakdown of 'make build' after core merge
-  #- make build SHELL=/bin/bash TAIL='|tail -40; exit $$PIPESTATUS'
-  - make assets
-  - make mo
-  - DJANGO_SETTINGS_MODULE=pootle.settings ./setup.py build_checks_templates
-  - ./manage.py migrate --noinput --traceback
-  - ./manage.py migrate | (egrep "Your models have changes" && exit 1 || exit 0)
-  - ./manage.py initdb --traceback --no-projects
-  - ./manage.py runserver --traceback &
-  - TESTPID=$!
-  - sleep 10
-  - kill -2 $TESTPID
+  - if [[ $META ]]; then python -m compileall -q -f .; fi
+  - if [[ $META ]]; then make docs; fi
+  - if [[ $META ]]; then make mo; fi
+  - if [[ $META ]]; then make pep8; fi
+  # - if [[ $META ]]; then pep257; fi
+  - if [[ ! $META ]]; then make assets; fi
+  - if [[ ! $META ]]; then DJANGO_SETTINGS_MODULE=pootle.settings ./setup.py build_checks_templates; fi
+  - if [[ ! $META ]]; then ./manage.py migrate --noinput --traceback; fi
+  - if [[ ! $META ]]; then ./manage.py migrate | (egrep "Your models have changes" && exit 1 || exit 0); fi
+  - if [[ ! $META ]]; then ./manage.py initdb --traceback --no-projects; fi
+  - if [[ ! $META ]]; then ./manage.py runserver --traceback; fi &
+  - if [[ ! $META ]]; then TESTPID=$!; fi
+  - if [[ ! $META ]]; then sleep 10; fi
+  - if [[ ! $META ]]; then kill -2 $TESTPID; fi
   # FIXME reenable migration testing
   #- tests/version-migration/migrate.sh
-  - make pep8
-  # - pep257
-  - py.test --cov=. -vv
+  - if [[ ! $META ]]; then py.test; fi
```

Comment:
the tests **do** use the database! 

also please dont get rid of cov or the -vv